### PR TITLE
Update Retroarch & Cores to be on par with V1.7.3 stable

### PR DESCRIPTION
**Pull request by Ntemis**
https://github.com/libretro/RetroArch/releases

### CODE REVIEW

File: packages/libretro/retroarch/package.mk [hunk 1]
```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="retroarch"
-PKG_VERSION="d360265"
+PKG_VERSION="91744d9"
```
Comment (RobLoach) [inline packages/libretro/retroarch/package.mk]: May or may not need to force a `--disable-qt`? Probably just sees Qt is not there, and then disables it automatically.

Comment (Ntemis) [reply packages/libretro/retroarch/package.mk]: https://github.com/libretro/RetroArch/commit/19f6798947f8a4dcbfcc43aab6d30686dfbf8e54

**Comment by gouchi** [comment]
We should consider to [update ScummVM core](https://github.com/libretro/scummvm/commits/master) as there are some "critical timing bug fix" and new feature "add MIDI support via 'libretro-deps/fluidsynth".

[Source](https://forums.libretro.com/t/scummvm-on-lakka-2-1-is-it-working/16104/7)

**Comment by ToKe79** [comment]
@Ntemis can you update `scummvm` to https://github.com/libretro/scummvm/commit/c65b4171b740d35da19d7ab8d37ee283e9b92aaa? I confirm that all [projects](http://nightly.builds.lakka.tv/special_builds/pr468/) are buildable, but I did not have time to test any image yet. However - this PR does not include the latest merge from LibreELEC 8.2. I will run another build with the latest commits in Lakka-2.1-dev.

**Pull request merged**